src/ResourceManager/Network/Commands.Network/help/Remove-AzureRmApplicationSecurityGroup.md

```diff
@@ -31,6 +31,19 @@ This command deletes an application security group named MyApplicationSecurityGr
 
 ## PARAMETERS
 
+### -AsJob
+Run cmdlet in the background```yaml
```

Comment:
Please move the yaml tag down to a new line to get the build to pass.  This is a known issue in platyPS that should be fixed soon.